woocommerce/woocommerce
# Rename @woocommerce/admin-library

### All Submissions:


### Changes proposed in this Pull Request:

In order to better identify the React Admin for WooCommerce Core, this pull request renames the package. We've settled on a `{plugin}/{category}/{component}` structure for these kinds of things in order to describe the project while also avoiding conflicts with any NPM or Composer packages in the monorepo.

Along that vein, this renames `@woocommerce/admin-library` to `woocommerce/client/admin`.

Closes #33407.

### How to test the changes in this Pull Request:

1. Run `pnpm -- turbo run build --filter=woocommerce/client/admin
2. Make sure the React Admin still works

### Other information:




### FOR PR REVIEWER ONLY:
